VEDA/settings.py

```diff
@@ -110,6 +110,15 @@
 # Python dotted path to the WSGI application used by Django's runserver.
 WSGI_APPLICATION = 'VEDA.wsgi.application'
 
+# See: http://getblimp.github.io/django-rest-framework-jwt/#additional-settings
+JWT_AUTH = {
+    # If the secret is wrong, JWT_VERIFY will raise a jwt.DecodeError telling you as such.
+    # You can still get at the payload by setting the JWT_VERIFY to False
+    'JWT_VERIFY': False,
```

Comment:
Please get rid of this. It should be se to `True` which is the default. If you don't verify the JWT, you are essentially opening the endpoint up to anyone.